code/framework/encryption/src/main/java/io/cattle/platform/framework/encryption/request/handler/DeTransformationHandler.java

```diff
@@ -0,0 +1,66 @@
+package io.cattle.platform.framework.encryption.request.handler;
+
+import io.github.ibuildthecloud.gdapi.context.ApiContext;
+import io.github.ibuildthecloud.gdapi.model.Field;
+import io.github.ibuildthecloud.gdapi.model.Resource;
+import io.github.ibuildthecloud.gdapi.model.Schema;
+import io.github.ibuildthecloud.gdapi.request.ApiRequest;
+import io.github.ibuildthecloud.gdapi.request.handler.AbstractApiRequestHandler;
+
+import java.io.IOException;
+import java.util.Map;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class DeTransformationHandler extends AbstractApiRequestHandler {
+
+    @Override
+    public void handle(ApiRequest request) throws IOException {
+        //TODO: remove this whole class after verifying that it works from wrapped resource.
```

Comment:
So i did but i guess i put it on the wrong pr.
It is on top of local auth pr. Ill fix that.